uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml

```diff
@@ -65,6 +65,7 @@ Page {
     id: popupView
 
     /*!
+       \deprecated
```

Comment:
Is this intentional?

Reply:
Yes, we plan on the deprecating popupManager with the release of the PopupElements toolkit components. 

Probably better to separate the depreciation of popupManager and the rest that goes with it into it's own PR. I'll take it out of this one.